poetry.toml

```diff
@@ -1,3 +1,3 @@
 [virtualenvs]
-create = true
+create = false
```

Comment:
If we're running outside docker we will want a virtual environment, so rather than setting this to false here we should just adjust it in our docker file, e.g. when running the install:
```
RUN poetry config virtualenvs.create false && poetry install
```

(Note that if we do, you'll need to adjust from watchmedo to `poetry run watchmedo` in your test entrypoint)